impacket/smbserver.py

```diff
@@ -2839,7 +2839,7 @@ def smb2SessionSetup(connId, smbServer, recvPacket):
                                                         authenticateMessage['domain_name'],
                                                         authenticateMessage['lanman'], authenticateMessage['ntlm'])
                     smbServer.log(ntlm_hash_data['hash_string'])
-                    if jtr_dump_path is not '':
+                    if jtr_dump_path != '':
```

Comment:
Referring to [PEP8](https://www.python.org/dev/peps/pep-0008/#programming-recommendations) "For sequences, (strings, lists, tuples), use the fact that empty sequences are false":
```suggestion
                    if jtr_dump_path:
```